test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -675,6 +677,184 @@ public void testGetDocIds() throws Exception {
   }
 
   @Test
+  public void testGetDocIdsActionColumnDelete() throws Exception {
+    executeUpdate("create table data(id integer, action varchar)");
+    executeUpdate("insert into data(id, action) values('1001', 'delete')");
+
+    Map<String, String> configEntries = new HashMap<String, String>();
+    configEntries.put("db.actionColumn", "action");
```

Comment:
We don't put something first just because it's under test elsewhere. This should be cleaned up, but either after uniqueKey or before metadataColumns makes more sense to me, especially the latter.